app/views/pages/moving-guide.html.erb

```diff
@@ -53,7 +53,7 @@
     <table>
       <tbody>
         <tr>
-          <th scope="row"><%= image_tag 'ppm.svg', alt: 'P P M', role: 'presentation' %><strong>Personally Procured Move</strong> (PPM)</th>
+          <th scope="row"><%= image_tag 'ppm.svg', role: 'presentation' %><strong>Personally Procured Move</strong> (PPM)</th>
```

Comment:
This change isn't exactly [what I suggested](https://github.com/deptofdefense/move.mil/pull/56#discussion_r133471680). The resulting markup from this change will be:

```html
<img role="presentation" src="/assets/ppm-45e7f24e6e6bb8e08531314089c4693603ca096785f37959532b582a3a24eabd.svg" alt="Ppm">
```

☝️ In the absence of `alt`, Rails' `image_tag` helper will assign a default `alt` attribute value of the name of the image. That's not what we want in this case.

The original feedback was to remove the text within the `alt` attribute, not remove the attribute entirely:

```erb
<%= image_tag 'ppm.svg', alt: '', role: 'presentation' %>
```